core/chains/terra/terratxm/orm_test.go

```diff
@@ -1,18 +1,27 @@
-package terratxm
+package terratxm_test
```

Comment:
is there a cycle causing this change?

Reply:
There at least _was_ but I'll check if it's still necessary. `.` import keeps it clean though FWIW.

Reply:
Still present.